mkosi/__init__.py

```diff
@@ -4214,7 +4215,7 @@ def ini_key_to_cli_arg(key: str) -> str:
                 continue
             # replace arguments referencing files with the file content
             try:
-                config = configparser.ConfigParser(delimiters="=")
+                config = configparser.RawConfigParser(delimiters="=")
```

Comment:
What's the motivation behind this change?

Reply:
The problem is that `ConfigParser` comes with its own interpolation facilities, that unfortunately clash with the format of systemd specifiers, because they too use %, e.g. `%(key)` expands to the value that key has if it was defined before in the config file. I also made a note of this in the commit message changing this.

Since @poettering noted in #626 that it would need some basic variable expansion, I went for `%u` to expand to user, so that specifier knowledge carries over from systemd. 

Unfortunately this change is backwards-incompatible, but I also don't think this is a terribly good feature to have for the mkosi config files as it was. That being said, I hope nobody uses this at the moment.

Reply:
I doubt it, LGTM